### PR TITLE
Shorten Neo Guide and align missing content with Readme

### PR DESCRIPTION
**Pull request by Ayelet41**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2,19 +2,19 @@
 
 ## Overview
 
-The Cloud Operations Sandbox is intended to make it easy for you to deploy and run a non-trivial application that lets you explore the Google Cloud Platform services, particularly the [Ops Management](http://cloud.google.com/products/operations) (formerly Stackdriver) product suite. Ops Management is a suite of tools that helps you gain full observability into your code and applications.
+The Cloud Operations Sandbox is intended to make it easy for you to deploy and run a non-trivial application that lets you explore the Google Cloud Platform services, particularly the [Cloud Operations](http://cloud.google.com/products/operations) (formerly Stackdriver) product suite. Cloud Operationsis a suite of tools that helps you gain full observability into your code and applications.
```
Comment (nicoleczhu) [inline docs/README.md:5]: add space `Operationsis` -> `Operations is`

File: docs/README.md [hunk 2]
```diff
@@ -2,19 +2,19 @@
...
 
 The Hipster Shop application used in the sandbox is intended to be sufficiently complex such that you can meaningfully experiment with it, and the Sandbox automatically provisions a new demo cluster, configures and deploys Hipster Shop, and simulates real users.
 
-With the Sandbox running, you can experiment with various Ops Management tools to solve problems and accomplish standard SRE tasks in a sandboxed environment without impacting your production monitoring setup.
+With the Sandbox running, you can experiment with various Cloud Operationstools to solve problems and accomplish standard SRE tasks in a sandboxed environment without impacting your production monitoring setup.
```
Comment (nicoleczhu) [inline docs/README.md:9]: `Cloud Operationstools` -> `Cloud Operations tools` -> 

Is your IDE getting formatted weirdly after you push to Github? This happened to me at one point...

File: docs/README.md [hunk 3]
```diff
@@ -2,19 +2,19 @@
 
 ## Overview
```
Comment (nicoleczhu) [inline docs/README.md:3]: Is a table of contents suitable here? If so, that might help the readability of this lengthy page. Suggestion, not a requirement :)

File: docs/README.md [hunk 4]
```diff
@@ -101,34 +101,40 @@ In a new browser tab, navigate to the Hipster Shop URL, where you can "purchase"
 ![image](./images/user-guide/2-hipstershop.png)
 
 ## Run the load generator
+Cloud Ops Sandbox comes with [Locust load generator](https://locust.io/), to simulate users traffic.  
 
-In another browser tab, navigate to the load-generator URL, from which you can simulate users interacting with the application to generate traffic. For this application, values like 100 total users with a spawn rate of 2 users per second are reasonable. Fill in the **Host** field with the "Hipster shop web address" from the installation stage if it isn't prepopulated. Click the **Start swarming** button to begin generating traffic to the site.
+- In another browser tab, navigate to the load-generator URL(from the installation stage if it isn't populated).
```
Comment (nicoleczhu) [inline docs/README.md:106]: Nit: `load-generator URL(from` -> `load-generator URL (from`

File: docs/README.md [hunk 5]
```diff
@@ -101,34 +101,40 @@ In a new browser tab, navigate to the Hipster Shop URL, where you can "purchase"
...
-# Learn Ops Management
+# Learn Cloud Operations
 
-## Ops Management Overview
+## Cloud Operations Overview
 
 As the cloud-native microservice architecture, which promises scalability and flexibility benefits, gets more popular, developers and administrators need tools that can work across cloud-based distributed systems.
 
-Ops Management provides products for both developers and administrators; this section introduces the products and their general audiences.  The tools are covered in more detail later.
+Cloud Operations provides products for both developers and administrators; this section introduces the products and their general audiences.  The tools are covered in more detail later.
```
Comment (nicoleczhu) [inline docs/README.md:123]: Nit: can use `.` instead of `;`

File: docs/README.md [hunk 6]
```diff
@@ -364,7 +370,7 @@ Search for the metric type using the name you gave it ("purchasing_counter_metri
 
 #### Logging Overview
 
-On detecting unusual symptoms in the charts, operators can look into Cloud Logging ([documentation](https://cloud.google.com/logging/docs/)) to find clues of it in the log messages. Filtering lets you identify relevant logs, and logs can be exported from Cloud Logging to "sinks" for long-term storage.
+On detecting unusual symptoms in the charts, operators can look into [Cloud Logging](https://console.cloud.google.com/logs)([documentation](https://cloud.google.com/logging/docs/)) to find clues of it in the log messages. Filtering lets you identify relevant logs, and logs can be exported from Cloud Logging to "sinks" for long-term storage.
```
Comment (nicoleczhu) [inline docs/README.md:373]: rephrase: `Operators can look at [logs](https://console.cloud.google.com/logs) in [Cloud Logging](https://cloud.google.com/logging/docs/) to find clues explaining any anomalies in the metrics charts.

File: docs/README.md [hunk 7]
```diff
@@ -413,7 +419,7 @@ Click **Create Sink**. Then follow the prompts to create a new storage bucket an
 
 #### Error Reporting Overview
 
-Cloud Error Reporting ([documentation](https://cloud.google.com/error-reporting/docs/)) automatically groups errors depending on the stack trace message patterns and shows the frequency of each error groups. The error groups are generated automatically, based on stack traces.
+[Cloud Error Reporting](https://console.cloud.google.com/errors) ([documentation](https://cloud.google.com/error-reporting/docs/)) automatically groups errors depending on the stack trace message patterns and shows the frequency of each error groups. The error groups are generated automatically, based on stack traces.
```
Comment (nicoleczhu) [inline docs/README.md:422]: `on stack trace message patterns and shows the frequency of each error group`

File: docs/tutorial.md [hunk 8]
```diff
@@ -10,11 +10,7 @@ With the Sandbox running, you can experiment with various Cloud Operations tools
 
 ### Architecture of the Hipster Shop application
 
-The Hipster Shop application consists of a number of microservices, written in a variety of languages, that talk to each other over gRPC.
-
-**Note:** We are not endorsing this architecture as the best way to build a real online store. This application is optimized for demonstration and learning purposes.  It illustrates a large number of cloud-native technologies, uses a variety of programming languages, and provides an environment that can be explored productively with Cloud Operations tools.
-
-The Git repository you cloned has all the source code, so you can explore the implementation details of the application. See the repository [README](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox) for a guided tour.
+The Hipster Shop application consists of a number of microservices, written in a variety of languages, that talk to each other over gRPC. [To learn more](<addlink to guide>
```
Comment (nicoleczhu) [inline docs/tutorial.md:13]: Do you need to add a link here?

File: docs/tutorial.md [hunk 9]
```diff
@@ -80,177 +71,43 @@ Cloud Ops Sandbox comes with [Locust load generator](https://locust.io/), to sim
...
+Navigate to the GCP Monitoring Workspace URL, which takes you to the [Cloud Monitoring **Workspace** page](https://console.cloud.google.com/monitoring) for your new project. The console may take some time to create a new workspace. Afterward, you'll be able to see a few dashboards generated through Cloud Operations tools.
 
 ## Learn Cloud Operations 
 
 ### Cloud Operations Overview
 
-As the cloud-native microservice architecture, which promises scalability and flexibility benefits, gets more popular, developers and administrators need tools that can work across cloud-based distributed systems.
-
-Cloud Operations provides products for both developers and administrators; this section introduces the products and their general audiences.  The tools are covered in more detail later.
+Cloud Operations provides products for both developers and administrators; this section introduces a few of the products, for additional information refer to the [User Guide] (<link>).
```
Comment (nicoleczhu) [inline docs/tutorial.md:84]: do you need to add a link here?

File: docs/tutorial.md [hunk 10]
```diff
@@ -80,177 +71,43 @@ Cloud Ops Sandbox comes with [Locust load generator](https://locust.io/), to sim
...
 
 -  [Cloud Trace](https://console.cloud.google.com/traces)
 -  [Cloud Profiler](https://console.cloud.google.com/profiler)
 -  [Cloud Debugger](https://console.cloud.google.com/debug)
 
-Similarly, cloud-native, microservice-based applications complicate traditional approaches used by administrators for monitoring system health: it's harder to observe your system health when the number of instances is flexible and the inter-dependencies among the many components are complicated. In the last few years, **Site Reliability Engineering (SRE)** has become recognized as a practical approach to managing large-scale, highly complex, distributed systems. Cloud Operations provides the following tools that are useful for SRE:
+Note: For application instrumentation your applications, the recommended solution is [**OpenCensus**](https://opencensus.io/), an open-source project that supports trace instrumentation in a variety of languages and that can export this data to Cloud.
```
Comment (nicoleczhu) [inline docs/tutorial.md:92]: What do you mean here: ` For application instrumentation your applications,`

File: docs/tutorial.md [hunk 11]
```diff
@@ -80,177 +71,43 @@ Cloud Ops Sandbox comes with [Locust load generator](https://locust.io/), to sim
...
+
+Similarly, cloud-native, microservice-based applications complicate traditional approaches used by administrators for monitoring system health: it's harder to observe your system health when the number of instances is flexible and the inter-dependencies among the many components are complicated. In the last few years, **Site Reliability Engineering (SRE)** has become recognized as a practical approach to managing large-scale, highly complex, distributed systems. 
+In addition, Cloud Operations provides the several tools that are useful for [Site Reliability Engineering (SRE)](<add link>):
```
Comment (nicoleczhu) [inline docs/tutorial.md:95]: Missing link?

File: docs/tutorial.md [hunk 12]
```diff
@@ -260,11 +117,9 @@ You can also use the Monitoring console to create alerts and uptime checks, and
 
 ### Monitoring and logs-based metrics
 
-Cloud Logging lets you define metrics based on information in structured logs. For example, you can count the number of log entries containing a particular message or extract latency info from log records. These "logs-based metrics" can then be charted with Cloud Monitoring. For details, see ["Using logs-based metrics"](https://cloud.google.com/logging/docs/logs-based-metrics/).
+Cloud Logging defines some logs-based metrics, but you can also create your own, for details, see ["Using logs-based metrics"](https://cloud.google.com/logging/docs/logs-based-metrics/).. To see the available metrics, select **Logging> Logs-based metrics** from the navigation panel. You see a summary of the system-provided and user-defined logs-based metrics.
```
Comment (nicoleczhu) [inline docs/tutorial.md:120]: rephrase: `Cloud Logging provides default, logs-based system metrics, but you can also create your own (see [Using logs-based metrics](https://cloud.google.com/logging/docs/logs-based-metrics/)). To see available metrics, select **Logging > Logs-based metrics** from the navigation panel. You should see both system metrics and some user-defined, logs-based metrics created in Sandbox.`

File: docs/tutorial.md [hunk 13]
```diff
@@ -292,9 +147,7 @@ Search for the metric type using the name you gave it.
 
 ## Cloud Logging
 
-### Logging Overview
-
-On detecting unusual symptoms in the charts, operators can look into [Cloud Logging](https://console.cloud.google.com/logs) to find clues of it in the log messages. Filtering lets you identify relevant logs, and logs can be exported from Cloud Logging to "sinks" for long-term storage.
+On detecting unusual symptoms in the charts, operators can look into [Cloud Logging](https://console.cloud.google.com/logs)([documentation](https://cloud.google.com/logging/docs/)) to find clues of it in the log messages. Filtering lets you identify relevant logs, and logs can be exported from Cloud Logging to "sinks" for long-term storage.
```
Comment (nicoleczhu) [inline docs/tutorial.md:150]: see rephrase suggestion above

File: docs/README.md (hunk 2 above)
Comment (Ayelet41) [reply docs/README.md:9]: How did you solve it?

File: docs/README.md (hunk 3 above)
Comment (Ayelet41) [reply docs/README.md:3]: This file is going to be deprecated(following PR) the reason I made the changes here is for alignment and reference (I also used it to compare between website -User Guide).

File: docs/README.md (hunk 4 above)
Comment (Ayelet41) [reply docs/README.md:106]: Changed it also in the website readme file

File: docs/README.md (hunk 2 above)
Comment (nicoleczhu) [reply docs/README.md:9]: I think I was using a different IDE. Now I use VScode with default settings.

**Review by Ayelet41 — COMMENTED** [review]
All changed were made

**Review by nicoleczhu — APPROVED** [review]

**Pull request merged**